app/views/components/Anchor.scala.html

```diff
@@ -14,8 +14,12 @@
  * limitations under the License.
  *@
 
-@()(implicit messages: Messages)
+@import config.FrontendAppConfig
+
+@this(appConfig: FrontendAppConfig)
+
+@(returnLocation: Option[String] = None)()
+
+<p class="govuk-body">
```

Comment:
I don't the changes here are correct. This component now has a pointless `returnLocation` parameter and an empty paragraph element here. Shouldn't this anchor component render a link?